client/styles/components/_overlay.scss

```diff
@@ -23,15 +23,22 @@
   flex-wrap: wrap;
   flex-flow: column;
   max-height: 80%;
+  max-width: 80%;
   position: relative;
   padding-bottom: #{25 / $base-font-size}rem;
+
+  @media (max-width: 650px) {
```

Comment:
My first instinct was that we should centralize these breakpoints into SCSS and JS variables.  Then I thought about it more and we don't necessarily need to use the same breakpoints for every component.  The point at which the overlay no longer fits might be different than the point where the nav bar doesn't fit, for example.  But if we want to add breakpoints to elements which are rendered as children of an overlay (like the preferences) then those would probably need to use the same breakpoints as the overlay.  Curious your thoughts on this.

Reply:
yeah I agree the breakpoints does not need to be same when they are completely unrelated like the navbar and the overlay